packages/core/tracing/src/components/trace-viewer/TraceViewer.vue

```diff
@@ -85,8 +105,9 @@ const { i18n: { t } } = composables.useI18n()
 
 const props = defineProps<{
   config: TraceViewerConfig
-  rootSpan: SpanNode
+  rootSpan?: SpanNode
   url?: string
+  skeleton?: boolean
```

Comment:
Could we rename this prop to `showSkeleton` or `isLoading`?

Reply:
Good idea. Updated